02-generalize/Square.cpp

```diff
@@ -0,0 +1,67 @@
+/*Program illustrates the drawing of a Square Box  in C++ programming*/
+
+#include <iostream>
+using namespace std;
+
+class Square
+{
+
+private:
+  int column,row,side;
+  char symbol;
+
+public:
+  void square_generate(char function_symbol,int function_side)
+  {
+    symbol = function_symbol;
+    side = function_side;
+
+    if(cin)
+    {
+      try
+      {
+        if(side < 1)
+        throw side;
+        else if(side == 1)
+        top_bottomSide(side);
+        else
+        square_draw(side);
+      }
+
+      catch(int errorVariable)
+      {
+        cerr << "Error: Expected Positive Integer Value For Box Side!\n";
```

Comment:
While writing error messages, always make the effort to write short, clear, simple, proper English. For instance, 
- Use proper caps (Consider: `"Error: Expected positive integer value For box side!\n"`)
- Make effort to write in least number of words (Consider: `"Error: Invalid input. Side should be positive integer."`)